denonavr/audyssey.py

```diff
@@ -64,8 +64,26 @@ def setup(self) -> None:
         for tag in self.appcommand0300_attrs:
             self._device.api.add_appcommand0300_update_tag(tag)
 
+        self._device.telnet_api.register_callback(
+            "PS", self._sound_detail_callback
+        )
+
         self._is_setup = True
 
+    def _sound_detail_callback(self, zone: str, event: str, parameter: str):
+        """Handle a sound detail change event."""
+        if self._device.zone == zone:
+            if parameter[0:6] == "REFLEV":
+                self._reflevoffset = parameter[7:]
+            elif parameter[0:6] == "DYNVOL":
+                self._dynamicvol = parameter[7:]
+            elif parameter[0:6] == "MULTEQ":
+                self._multeq = parameter[7:]
+            elif parameter == "DYNEQ ON":
+                self._dynamiceq = "1"
+            elif parameter == "DYNEQ OFF":
+                self._dynamiceq = "0"
```

Comment:
```suggestion
        if self._device.zone != zone:
            return

        if parameter[0:6] == "REFLEV":
            self._reflevoffset = parameter[7:]
        elif parameter[0:6] == "DYNVOL":
            self._dynamicvol = parameter[7:]
        elif parameter[0:6] == "MULTEQ":
            self._multeq = parameter[7:]
        elif parameter == "DYNEQ ON":
            self._dynamiceq = "1"
        elif parameter == "DYNEQ OFF":
            self._dynamiceq = "0"
```
The same for the other callbacks where `zone` is checked.